### DISCUSSION TITLE
HA Tasmota Auto-discovery does not work with tasmota-display.bin

### DISCUSSION THREAD

**Discussion by ghost** [arendst/Tasmota#12663] [Q&A] [answered]
I am not sure if this is intended, so I am posting here instead of the **Issues** report.

I am using tasmota-display.bin from v9.5.0.1 dev, and also v9.5.0 rel. Both binaries did not provide HA auto discovery using Tasmota integration. When switched to tasmota.bin or tasmota-sensors.bin, they are auto discovered.

I checked the MQTT entries using MQTT Explorer, and noticed that tasmota-display.bin does not make entry at tasmota/discovery topic.

Is this as designed? Any way to enable it when self-compiling?

**Comment by sfromis** [comment, marked as answer]
You have
`#define USE_TASMOTA_DISCOVERY                      // Enable Tasmota Discovery support`

It is related to `USE_HOME_ASSISTANT` which is by design not included in `tasmota-display.bin`
Note that if you only use the new Tasmota integration in HA, you do not need to have the latter option enabled.
